src/filter/distort0r/distort0r.c

```diff
@@ -179,66 +212,66 @@ void interpolateGrid(grid_point_t* grid, unsigned int w, unsigned int h,
-	    >> GRID_SIZE_LOG; 
-	  int32_t step_end_col_u   = (lower_right->u - upper_right->u) 
-	    >> GRID_SIZE_LOG; 
-	  int32_t step_end_col_v   = (lower_right->v - upper_right->v) 
-	    >> GRID_SIZE_LOG; 
-	  
-	  int32_t u_line_index, v_line_index; 
-	  int32_t step_line_u, step_line_v; 
-	  
-	  
+	  unsigned int offset = x + y*(grid_x+1);
+
+	  grid_point_t* upper_left  = grid + offset;
+	  grid_point_t* lower_left  = grid + offset + grid_x + 1;
+	  grid_point_t* upper_right = grid + offset + 1;
+	  grid_point_t* lower_right = grid + offset + grid_x + 2;
+
+	  int32_t u_left, u_right, v_left, v_right;
+
+	  int32_t start_col_uu = upper_left->u;
+	  int32_t start_col_vv = upper_left->v;
+	  int32_t end_col_uu   = upper_right->u;
+	  int32_t end_col_vv   = upper_right->v;
+
+	  int32_t step_start_col_u = (lower_left->u - upper_left->u)
+	    >> GRID_SIZE_LOG;
+	  int32_t step_start_col_v = (lower_left->v - upper_left->v)
+	    >> GRID_SIZE_LOG;
+	  int32_t step_end_col_u   = (lower_right->u - upper_right->u)
+	    >> GRID_SIZE_LOG;
+	  int32_t step_end_col_v   = (lower_right->v - upper_right->v)
+	    >> GRID_SIZE_LOG;
+
+	  int32_t u_line_index, v_line_index;
+	  int32_t step_line_u, step_line_v;
+
+
 	  uint32_t* pos = dst+ (y<<GRID_SIZE_LOG)*w + (x<<GRID_SIZE_LOG);
-      
-	  for(block_y = 0; block_y < GRID_SIZE; ++block_y) 
-	    { 
-	      u_left  = start_col_uu; 
-	      u_right = end_col_uu; 
-	      v_left  = start_col_vv; 
-	      v_right = end_col_vv; 
-	      
-	      u_line_index = start_col_uu; 
-	      v_line_index = start_col_vv; 
-	      
-	      step_line_u = (int32_t) ((u_right-u_left) >> GRID_SIZE_LOG); 
-	      step_line_v = (int32_t) ((v_right-v_left) >> GRID_SIZE_LOG); 
-	      
-	      for(block_x=0; block_x < GRID_SIZE; ++block_x) 
-		{ 
+
+	  for(block_y = 0; block_y < GRID_SIZE; ++block_y)
+		{
+		  u_left  = start_col_uu;
+		  u_right = end_col_uu;
+		  v_left  = start_col_vv;
+		  v_right = end_col_vv;
+
+		  u_line_index = start_col_uu;
+		  v_line_index = start_col_vv;
+
+		  step_line_u = (int32_t) ((u_right-u_left) >> GRID_SIZE_LOG);
+		  step_line_v = (int32_t) ((v_right-v_left) >> GRID_SIZE_LOG);
+
+		  for(block_x=0; block_x < GRID_SIZE; ++block_x)
+		{
 		  int uu = u_line_index >> 16;
 		  int vv = v_line_index >> 16;
-		  
-		  u_line_index += step_line_u; 
-		  v_line_index += step_line_v; 
-		  
-		  *pos++ = src[uu + vv * w]; 
-		} 
-	      
-	      start_col_uu += step_start_col_u; 
-	      end_col_uu   += step_end_col_u; 
-	      start_col_vv += step_start_col_v; 
-	      end_col_vv   += step_end_col_v; 
-	      
-	      pos += (w - GRID_SIZE); 
-	    }   
+
+		  u_line_index += step_line_u;
+		  v_line_index += step_line_v;
+
+		  *pos++ = src[uu + vv * w];
+		}
+
+		  start_col_uu += step_start_col_u;
+		  end_col_uu   += step_end_col_u;
+		  start_col_vv += step_start_col_v;
+		  end_col_vv   += step_end_col_v;
+
+		  pos += (w - GRID_SIZE);
+		}
+	}
 	}
-    }
```

Comment:
This last hunk I believe is purely whitespace. Please do not include superfluous changes.